src/releng_docs/services_shipit.rst

```diff
@@ -3,53 +3,25 @@
 Service family: ``src/shipit_*``
```

Comment:
Yes, a bit of formatting and I removed obsolete documentation. I'm preparinng docs to be able to document ``please`` command.